packages/plugin-commands-publishing/src/publish.ts

```diff
@@ -102,7 +102,7 @@ export async function handler (
         hint: GIT_CHECKS_HINT,
       })
     }
-    const branch = opts.publishBranch ?? 'master'
+    const branch = opts.publishBranch ?? (await getDefaultBranch() || 'main')
```

Comment:
let's make it an array and allow both main and master. In that case, it won't be a breaking change